src/features/ChangePassword/ui/ChangePasswordForm.tsx

```diff
@@ -61,6 +61,8 @@ export const ChangePasswordForm = ({ title, token, email, onSuccessExtended }: C
   const approveErrorMessage = useMemo(() => approveError?.evaluatedMessage, [approveError])
   const updateErrorMessage = useMemo(() => updateError?.evaluatedMessage, [updateError])
 
+  const approveSuccessMessage = useMemo(() => (isApproveSuccess ? t("success") : null), [isApproveSuccess, t])
```

Comment:
I didn't notice that during previous reviews but anyway: You don't need `useMemo` here. If it is a dead simple computation then we don't wrap it in useMemo

Reply:
Done